src/index.ts

```diff
@@ -0,0 +1,78 @@
+import * as capitano from 'capitano';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+import JSZip from 'jszip';
+import * as Path from 'path';
+import * as skhema from 'skhema';
+
+const specPath: string = '/source/specification/spec.json';
+const zip =  new JSZip();
+
+const generate = async (
+	params: { folder: string },
+	options: { output: string },
+) => {
+
+	const rfq = await validateSpec(params.folder + specPath); // generate rfq json from spec.json and schema
+	zip.file('rfq.json', JSON.stringify(rfq, null, 2)); // add rfq.json to the release
+
+	const testFile = fs.readFileSync(Path.join(params.folder, 'testing', 'Testing.md'));
+	const testContent = Buffer.from(testFile).toString('utf8');
+	zip.file('Testing.md', testContent);
+
+	packageFiles(Path.join(params.folder, 'outputs'));
+
+	const data = await zip.generateAsync({
+		type: 'nodebuffer',
+		compression: 'DEFLATE'
+	});
+	fs.writeFileSync(Path.join(options.output, `release.zip`), data, 'binary');
+};
+
+// check the spec file against schema for the project type - this will have to change if we get rid of the spec.json file from the repos
+const validateSpec = async(spec: string) => {
+	let specFile = JSON.parse(fs.readFileSync(spec).toString('utf8'));
+	const schema = await JSON.parse(fs.readFileSync(Path.join(__dirname,'../', 'hardware-types', specFile.hwType, 'rfq.json')).toString('utf8'));
+	const partRFQ = skhema.filter(schema, specFile); // this could be a check rather than a filter
+	return partRFQ;
+};
+
+
+// get all files in output folder - should we restrict the file types to get depending on project type
+const packageFiles = async(path: string) => {
+	fs.readdirSync(path, {withFileTypes: true}).forEach(fileName => { // nead this to be recursive
+		console.log(path);
+		console.log(fileName.name);
+		// if its a directory, do it again - if its a file, zip it
+		if(fileName.isDirectory()){
+			console.log('this is a directory')
+			packageFiles(Path.join(path, fileName.name));
+		} else {
+			console.log('this is a file')
+			const fileRead = fs.readFileSync(Path.join(path, fileName.name))
+			const fileContent = Buffer.from(fileRead).toString('utf8');
+			zip.file(fileName.name, fileContent) // what should the files get called? is it ok just to dump all the files in there?
```

Comment:
@rcooke-warwick Should we take this as an input from capitano? I think it makes sense to accept this as branch from which release is created or even better the subject of the email.

Reply:
@anujdeshpande if/when we moved it into versionbot then I agree, it will be taken from the cloned branch.

Reply:
`Moving this to versionbot` seems like something we can discuss in today's call then